tests/unit/session-stores/shared/cookie-store-behavior.js

```diff
@@ -39,11 +39,12 @@ export default function(options) {
 
   describe('#persist', function() {
     beforeEach(function() {
-      sinon.spy(CookieSessionStore.prototype, '_warn');
+      sinon.spy(Ember, 'warn');
```

Comment:
Hmm. This is what `Ember.Debug.registerWarnHandler` is for..

Reply:
Check out:

https://github.com/emberjs/ember-test-helpers/blob/4fbc32f53c221cc4ac709ae9b99095197d6e3282/tests/test-helper.js#L37-L50

And then:

https://github.com/emberjs/ember-test-helpers/blob/4fbc32f53c221cc4ac709ae9b99095197d6e3282/tests/test-helper.js#L77-L103

(`registerDeprecationHandler` is essentially the same as `registerWarnHandler`)

Reply:
does that work all the way down to Ember 1.12? (see https://github.com/simplabs/ember-simple-auth/blob/master/package.json#L99)

Reply:
Yes, with the polyfill in your devDeps

Reply:
https://github.com/rwjblue/ember-debug-handlers-polyfill

Reply:
@rwjblue looks like that polyfill only supports `registerDeprecationHandler`, but not `registerWarnHandler` 🤔